rules/line-between-sibling-blocks.js

```diff
@@ -0,0 +1,108 @@
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'line-between-sibling-blocks';
+    }
+
+    test(data) {
+        const { formatMessage, addError } = this.outcome;
+        let currentIndex = 0;
+        const contentParser = content => {
+            const blockRegExp = /{%[' ']{0,}(block|widget|embed)[a-zA-Z0-9 _\.\[\]\':\{\}\-\~\,\(\)\?\n\r]{1,}%}/igm;
+            const includingRegExp = /{%[' ']{0,}include[a-zA-Z0-9 _\.\[\]\':\{\}\-\~\,\(\)\?\n\r]{1,}%}/igm;
+            const closingTagRegExp = /{%[' ']{0,}(endblock|endwidget|endembed)[' ']{0,}%}/igm;
+            const blockTags = content.match(blockRegExp);
+            const includes = content.match(includingRegExp);
+            const closingTags = content.match(closingTagRegExp);
+            const blockTagsData = blockTags ? organizeData(content, blockTags) : null;
+            const includesData = includes ? organizeData(content, includes) : null;
+            const closingTagsData = closingTags ? organizeData(content, closingTags) : null;
+
+            return {
+                blockTagsData: blockTagsData,
+                includesData: includesData,
+                closingTagsData: closingTagsData,
+            }
+        };
+
+        const findEntryIndex = (startIndex, content, entry) => {
+            const index = content.slice(startIndex).indexOf(entry);
+
+            return startIndex + index;
+        };
+
+        const organizeData = (content, tags) => tags.map((tag, index, arr) => {
+                const entryIndex = findEntryIndex(currentIndex, content, tag);
+                currentIndex = entryIndex + tag.length;
+
+                if (index === arr.length - 1) {
+                    currentIndex = 0;
+                }
+
+                return {
+                    data: tag,
+                    index: entryIndex,
+                }
+            });
+
+        this.filterModulesData(data, 'twig').forEach(twigFileData => {
+            const { files, type, name } = twigFileData;
+            const { twig } = files;
+            const { path, api } = twig;
+            const { external } = api;
+            const blockNames = (external && external.blocks) ? external.blocks : null;
+            const getValidationString = (closingTagData, nextBlockData) => twig.content.slice(closingTagData.index + closingTagData.data.length, nextBlockData.index);
+
```

Comment:
Please write `twig.content......` from the new line, max string length should be 120 characters